tests/headers/attributes.hpp

```diff
@@ -1,3 +1,6 @@
+#include <stdint.h>
+#include <stddef.h>
```

Comment:
Don't include system headers in tests since bindgen will produce different output depending on the content of them, so tests will never pass. If you definitely need them, add another file to the `bindgen-integration` crate.